### PR TITLE
Update project-generation.md

### PR DESCRIPTION
**Pull request by slebed**
Initial rewrite to update using Gdx-LiftOff.

TODO: There are comments that will need to be cleaned up before merging.

### CODE REVIEW

File: wiki/articles/updating-libgdx.md [hunk 1]
```diff
@@ -66,4 +66,4 @@ As an alternative, you can specify a Gradle distribution by URL (take a look [he
 ```
 
 ## Additional Steps
-Since Gradle updates often introduce breaking changes, you might need to take additional steps to get your project running again after an update. Usually, we recommend just recreating your project structure with the setup tool and then copying over your dependencies and code. Alternatively, you can take a look at the changes we made to the setup tool's example project [here](https://github.com/libgdx/libgdx/commits/master/extensions/gdx-setup).
+Since Gradle updates often introduce breaking changes, you might need to take additional steps to get your project running again after an update. Usually, we recommend just recreating your project structure with the gdx-liftoff setup tool and then copying over your dependencies and code. <!--Alternatively, you can take a look at the changes we made to the setup tool's example project [here](https://github.com/libgdx/libgdx/commits/master/extensions/gdx-setup).-->
```
Comment (tommyettinger) [inline wiki/articles/updating-libgdx.md]: The commented section could be replaced to mention the already-generated sample repos: https://github.com/libgdx/gdx-liftoff-demo and/or https://github.com/libgdx/gdx-liftoff-demo-kotlin . Those do have a history that gets updated generally once per release, which makes it easy enough to see what changed.

Comment (slebed) [reply wiki/articles/updating-libgdx.md]: Added the link to the libgdx demo

File: wiki/articles/updating-libgdx.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Since version 1.9.13, breaking changes and corresponding migration steps are exp
 
 #### Update to release 1.9.12
 
-* HTML: You can delete the soundmanager files for HTML Project and the references in index.html
+* HTML: You can delete the soundmanager files for HTML Project and the references iupdated pages replacing gdx-setup with gdx-liftoffn index.html
```
Comment (Frosty-J) [inline wiki/articles/updating-libgdx.md:38]: Take a closer look at your remnants of `in`.

**Comment by raeleus** [comment]
@slebed, I have continued this work in #198. Thank you very much for getting this started!

**Pull request closed without merging**